core-bundle/src/DependencyInjection/Compiler/AddPackagesPass.php

```diff
@@ -12,24 +12,30 @@
 
 namespace Contao\CoreBundle\DependencyInjection\Compiler;
 
-use Contao\CoreBundle\Util\PackageUtil;
-use PackageVersions\Versions;
+use Composer\InstalledVersions;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
 /**
  * Adds the composer packages and version numbers to the container.
  *
  * @internal
+ *
+ * @deprecated Deprecated since Contao 4.5, to be removed in Contao 5.0; use
+ *             the PackageUtil class instead
```

Comment:
that's probably wrong?

Reply:
Good catch. Fixed in 415922e3d5f5be3a6b259ebfa653a5a154e6e6bc.